pkg/api/types.go

```diff
@@ -2669,6 +2673,69 @@ type SecurityContext struct {
 	ReadOnlyRootFilesystem *bool `json:"readOnlyRootFilesystem,omitempty"`
 }
 
+// SeccompProfile represents a seccomp profile. A profile is composed of a list of seccomp
+// rules and a default action. Seccomp profiles are unnamespaced.
+type SeccompProfile struct {
+	unversioned.TypeMeta `json:",inline"`
```

Comment:
Does this mean I should put it in another file?

On Monday, May 9, 2016, Paul Morie <notifications@github.com> wrote:

> In pkg/api/types.go
> <https://github.com/kubernetes/kubernetes/pull/25324#discussion_r62506322>
> :
>
> > @@ -2669,6 +2673,69 @@ type SecurityContext struct {
> >  	ReadOnlyRootFilesystem *bool `json:"readOnlyRootFilesystem,omitempty"`
> >  }
> >
> > +// SeccompProfile represents a seccomp profile. A profile is composed of a list of seccomp
> > +// rules and a default action. Seccomp profiles are unnamespaced.
> > +type SeccompProfile struct {
> > +	unversioned.TypeMeta `json:",inline"`
>
> Yep, we need it here. SeccompProfile will be its own API resource.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/25324/files/ec54b7e13bbd79087fb34f0973f4482c0150c9c6#r62506322>
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>